src/index.tsx

```diff
@@ -444,8 +444,8 @@ export default class ReactToPrint extends React.Component<IReactToPrintProps> {
                             // the current page. Some will attempt to load the page"s parent
                             // directory. These problems can cause `react-to-print` to stop without
                             // any error being thrown. To avoid such problems we simply do not
-                            // attempt to load these links.
-                            if (node.getAttribute("href")) {
+                            // attempt to load these links. 
+                            if (node.getAttribute("href") && !node.hasAttribute("disabled")) {
```

Comment:
any link node with `disabled`attribute, regardless of any of the following, will not trigger `onload` or `onerror`
 callback. We should skip them.
```
<link disabled ... /> 
<link disabled="true" ... />`
<link disabled="false" ... />` 
```